src/styles/blocks/main.scss

```diff
@@ -0,0 +1,28 @@
+main {
+  padding: 50px 40px;
```

Comment:
According to the code style rules, there should not be a space before the semicolon.